src/views/SafeAreaView.js

```diff
@@ -153,7 +150,19 @@ class SafeView extends Component {
     const { touchesTop, touchesBottom, touchesLeft, touchesRight } = this.state;
     const { forceInset, isLandscape } = this.props;
 
+    const {
```

Comment:
would be good to have a comment about this destructuring block to clarify why we're setting the default values like this

Reply:
added